### DISCUSSION TITLE
Hackathon: 2024-03-19

### DISCUSSION THREAD

**Discussion by stefaniebutland** [earthaccess-dev/earthaccess#496] [Hack days]
## Reporting out from earthaccess hackday Mar 19, 2024

Please create a new comment at the bottom to write about what you did today! Include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by danielfromearth** [comment]
@mfisher87, @chuckwondo, and myself worked more on a unit test for the [UAT environment access PR #426](https://github.com/nsidc/earthaccess/pull/426), specifically this [commit](https://github.com/nsidc/earthaccess/pull/426/commits/301568c94234a10bd2498a215f37da597d2dcca4).  Also spent time debugging a development environment error being raised during `poetry install`.  Resolved it by removing the poetry and conda environments and reinstalling the environments.

**Comment by andypbarrett** [comment]
Worked on reorganizing docs.  Focused on index.md to give a simple getting started example.  See https://github.com/nsidc/earthaccess/tree/update-documentation

**Comment by mfisher87** [comment]
@doug-newman-nasa gave a fantastic (recorded) presentation on unit testing best practices and the VCR unit testing method using `vcrpy`. We'll post the recording URL here once it's online! @asteiker can I help with that?

**Reply by stefaniebutland** [reply]
upload to Openscapes YouTube > NASA-Openscapes playlist in progress (1:50pm PT). Will appear at https://youtu.be/u3R2fwwNdVo

**Comment by itcarroll** [comment]
@itcarroll (slowly re-waking from vacation) returned to working on #486.

**Reply by mfisher87** [reply]
Welcome back :)

**Reply by jhkennedy** [reply]
@itcarroll I'm also back from vacation -- let me know if you want to meet up to talk about #486, or work on it next hackday together.
